wgsl/index.bs

```diff
@@ -522,6 +540,25 @@ or a [=hexadecimal floating point literal=].
     * The value of the literal is the value of the mantissa multiplied by 2 to the power of the exponent.
          When no exponent is specified, an exponent of 0 is assumed.
 
+<div class='example wgsl float-literals' heading='floating point literals'>
+  <xmp highlight='rust'>
+    const a = 0.e+4f;
+    const b = 01.;
+    const c = .01;
+    const d = 12.34;
+    const f = .0f;
+    const g = 0h;
+    const h = 1e-3;
+    const i = 0xa.fp+2;
+    const j = 0x1P+4f;
+    const k = 0X.3;
+    const l = 0x3f;
```

Comment:
This is not a float literal, f is a valid hex numeric char

Reply:
Hah, good point, will move. The lack of `f` on the end of hex floats keeps throwing me off.

Reply:
Thanks for the fix!